runtime/port/win32/j9shsem.c

```diff
@@ -176,10 +214,10 @@ j9shsem_open(struct J9PortLibrary *portLibrary, struct j9shsem_handle **handle,
   
 		if (shsem_handle->mainLock == NULL) {
 			Trc_PRT_shsem_j9shsem_open_Event2(semaphoreName);
-			rc = createSemaphore(portLibrary, shsem_handle);
+			rc = createSemaphore(portLibrary, shsem_handle, global);
```

Comment:
If the semaphore is being created, shouldn't it always be the global semaphore? I'm thinking there should be an assert to validate this.

Reply:
> If the semaphore is being created, shouldn't it always be the global semaphore?

Not if it is an IBM JVM.  See https://github.com/eclipse/openj9/pull/5613#issuecomment-492289205.

Reply:
That comment was made in a different context. As long as there is no backwards compatibility concern, it doesn't need to be restricted to OpenJ9.

Reply:
Sorry, I misinterpreted the comment.